src/Render/Element/Viewfield.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Core\Render\Element\Viewfield.
+ */
+
+namespace Drupal\Core\Render\Element;
+
+use Drupal\Core\Render\Element;
+
+/**
+ * Renders a view of the selected view.
+ *
+ * @RenderElement("viewfield")
+ */
+class Viewfield extends RenderElement {
+
+  public function getInfo() {
```

Comment:
This needs at least function documentation. I guess `{@inheritdoc}` should do its job here.